lib/renderers/0-3.js

```diff
@@ -174,7 +177,7 @@ export default class Renderer {
         if (isValidMarkerType(tagName)) {
```

Comment:
I'm not overly concerned as for 1.0 I'd like to drop the legacy 0.2 renderer. Probably once I ensure there is an ok path to upgrading existing documents though.